app/javascript/components/OrderManagerFilesetsForm.vue

```diff
@@ -159,15 +136,33 @@ export default {
   },
   watch: {
     method: function (val) {
-      if (val === 'foliate') {
-        this.unitLabel = 'f. '
-      } else if (val === 'paginate') {
-        this.unitLabel = 'p. '
-      }
       this.updateMultiLabels()
     }
   },
   methods: {
+    labelerOpts () {
```

Comment:
I really did *not* want to introduce this change, but I could not find a way to ensure that `labelerOpts` as a computed property flushed cached values when `this.unitLabel` was updated elsewhere (which should happen given my understanding of https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods)